recipe/conda_build_config.yaml

```diff
@@ -1,54 +1,3 @@
-c_compiler_version: # works in conjunction with c_stdlib_version
-  - 12          # [linux and aarch64]
-  - 12          # [linux and x86_64]
-  - 11          # [linux and aarch64]
-  - 11          # [linux and x86_64]
-
-cxx_compiler_version:
-  - 12          # [linux and aarch64]
-  - 12          # [linux and x86_64]
-  - 11          # [linux and aarch64]
-  - 11          # [linux and x86_64]
-
-c_stdlib_version:
-  - 2.28        # [linux and aarch64]
-  - 2.17        # [linux and x86_64]
-  - 2.17        # [linux and aarch64]
-  - 2.17        # [linux and x86_64]
-
-fortran_compiler_version:
-  - 12          # [linux and aarch64]
-  - 12          # [linux and x86_64]
-  - 11          # [linux and aarch64]
-  - 11          # [linux and x86_64]
-
-cuda_compiler_version:
-  - 12.0        # [linux and aarch64]
-  - 12.0        # [linux and x86_64]
-  - 12.0        # [win]
-  - 11.8        # [linux and aarch64]
-  - 11.8        # [linux and x86_64]
-  - 11.8        # [win]
-
-cuda_compiler:
-  - cuda-nvcc   # [linux and aarch64]
-  - cuda-nvcc   # [linux and x86_64]
-  - cuda-nvcc   # [win]
-  - nvcc        # [linux and aarch64]
-  - nvcc        # [linux and x86_64]
-  - nvcc        # [win]
-
-cdt_name:
-  - cos7        # [linux and aarch64]
-  - cos7        # [linux and x86_64]
-  - cos7        # [linux and aarch64]
-  - cos7        # [linux and x86_64]
-
-docker_image:
-  # TODO: We're using the aarch64-cuda:11.8 image, based on ubi8, only to get a
-  # newer glibc for the CUDA 12 builds. For some reason, aarch64 + CUDA 12 is
-  # expecting GLIBC_2.27.
-  - quay.io/condaforge/linux-anvil-aarch64-cuda:11.8  # [linux and aarch64] # glibc 2.28
-  - quay.io/condaforge/linux-anvil-cos7-x86_64        # [linux and x86_64]  # glibc 2.17
-  - quay.io/condaforge/linux-anvil-aarch64-cuda:11.8  # [linux and aarch64] # glibc 2.28
-  - quay.io/condaforge/linux-anvil-cuda:11.8          # [linux and x86_64]  # glibc 2.17
+# https://docs.nvidia.com/deeplearning/cudnn/v9.3.0/reference/support-matrix.html#cpu-architecture-and-os-requirements
+c_stdlib_version:  # [linux]
+  - 2.28  # [linux]
```

Comment:
Note that I changed the minimum glibc version to 2.28 for all archs because of the release notes, but I'm not sure that it's strictly true that you can't run on 2.17.